src/bootstrap/lib.rs

```diff
@@ -923,6 +923,21 @@ impl Build {
         self.config.use_lld && !target.contains("msvc")
     }
 
+    fn lld_flags(&self, target: TargetSelection) -> impl Iterator<Item = String> {
+        let mut options = [None, None];
+
+        if self.config.use_lld {
+            if self.is_fuse_ld_lld(target) {
+                options[0] = Some("-Clink-arg=-fuse-ld=lld".to_string());
+            }
+
+            let threads = if target.contains("windows") { "/threads:1" } else { "--threads=1" };
```

Comment:
Ditto

Reply:
This was introduced in #84865, which claims the opposite. I only modified it.

Reply:
Oh, so there are 2 bugs.
I'll prepare the patch for `--threads` and ask the maintainer about `/<option>:no` because it doesn't look like intended.

Please disregard my comments then.